packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -150,7 +196,8 @@ class ServiceConnectionManager extends ServiceManager<VmServiceWrapper> {
   ///
   /// Throws an Exception if no 'FlutterView' is present in this isolate.
   Future<String> get flutterViewId async {
```

Comment:
superficially extensions like this seem equally useful for devtools extensions as for devtools proper.

Reply:
Agreed we may want to expose this. For now, I'd like to keep the API small and add more to `devtools_app_shared` as it makes sense.